zjit.rb

```diff
@@ -242,7 +245,7 @@ def print_counters(keys, buf:, stats:, right_align: false, base: nil)
       case key
       when :ratio_in_zjit
         value = '%0.1f%%' % value
-      when :guard_type_exit_ratio
+      when :guard_type_exit_ratio, :guard_shape_exit_ratio
```

Comment:
Can we merge this `when` with the above `when :ratio_in_zjit`? I couldn't figure out why they're separate.